spock-core/src/main/java/org/spockframework/util/VersionChecker.java

```diff
@@ -25,7 +25,7 @@
   public void checkGroovyVersion(String whoIsChecking) {
     if (!isCompatibleGroovyVersion()) {
       if (isVersionCheckDisabled()) {
-        System.err.println(format("Executing Spock %s with NOT compatible Groovy version %s due to set %s system property set. " +
+        System.err.println(format("Executing Spock %s with NOT compatible Groovy version %s due to set %s system property. " +
```

Comment:
No related change that I already had locally. I might do a separate PR if you insist ;).